tests/test_utils.py

```diff
@@ -875,6 +875,17 @@ def test_argument_none_error(self):
         ):
             pu.inv(None)
 
+    def test_callable_argument_error(self):
+        """Test that the proper error is raised when the argument is a function."""
+        def func(x):
+            return x
+
+        with pytest.raises(
+            ValueError,
+            match="A function was passed as an argument to inv. ",
+        ):
```

Comment:
Black is great except for those 1% of times when it makes things really ugly :slightly_frowning_face: